sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -9,31 +11,48 @@ Software requirements:
 Installing OpenCL 2.1 compatible software stack:
 1. OpenCL headers:
 
-   a. Download the OpenCL headers from [github.com/KhronosGroup/OpenCL-Headers](https://github.com/KhronosGroup/OpenCL-Headers) to your local machine. e.g. `/usr/local/include/CL` with environment var `$OPENCL_HEADERS`.
+   a. Download the OpenCL headers from
+[github.com/KhronosGroup/OpenCL-Headers](https://github.com/KhronosGroup/OpenCL-Headers)
+to your local machine. e.g. `/usr/local/include/CL` with environment var
+`$OPENCL_HEADERS`.
 2. OpenCL runtime for CPU and GPU:
 
-   a. OpenCL runtime for GPU: follow instructions on [github.com/intel/compute-runtime/releases](https://github.com/intel/compute-runtime/releases) to install.
+   a. OpenCL runtime for GPU: follow instructions on
+[github.com/intel/compute-runtime/releases](https://github.com/intel/compute-runtime/releases)
+to install.
 
-   b. OpenCL runtime for CPU: follow instructions under section "Intel&reg; CPU Runtime for OpenCL. Applications 18.1 for Linux* OS (64bit only)" on [https://software.intel.com/en-us/articles/opencl-drivers#cpu-section](https://software.intel.com/en-us/articles/opencl-drivers#cpu-section) and click on orange "Download" button to download & install.
+   b. OpenCL runtime for CPU: follow instructions under section "Intel&reg; CPU
+Runtime for OpenCL. Applications 18.1 for Linux* OS (64bit only)" on
+[https://software.intel.com/en-us/articles/opencl-drivers#cpu-section](https://software.intel.com/en-us/articles/opencl-drivers#cpu-section)
+and click on orange "Download" button to download & install.
 
 # Build the SYCL compiler
 
-Download the LLVM* repository with SYCL support to your local machine folder e.g. `$HOME/sycl` (assuming environment var `$SYCL_HOME`) folder using following command:
+Download the LLVM* repository with SYCL support to your local machine folder
+e.g. `$HOME/sycl` (assuming environment var `$SYCL_HOME`) folder using
+following command:
 
 ```
 git clone https://github.com/intel/llvm -b sycl $HOME/sycl
 ```
```

Comment:
can we use bash syntax specified for bash code blocks

Reply:
Done.